### PR TITLE
First Desktop version portfolio

### PR DESCRIPTION
**Pull request by osmanwako**
-  Use mobile version for 768px device
-  four sections used a grid layout
- inner section use flex blox
-  no functionality that requires JavaScript.
- font, colors, images, text, and margins are depending on Figma design

### CODE REVIEW

File: assets/style/main.css [hunk 1]
```diff
@@ -0,0 +1,517 @@
...
+  background-repeat: no-repeat;
+  background-position: 0 0;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border-bottom-left-radius: 15%;
+  margin-bottom: 40px;
+}
```
Comment (Tufahel) [inline assets/style/main.css]: - [ ] Kindly consider not using `background-size: cover` for this section, cause for this reason it loses its responsiveness that's why it is vanishing behind the windows. I suggest you use `background-size: x% y%;`.

***In the video I have suggested an implementation, you consider it***
ezgif com-video-to-gif

File: assets/style/main.css [hunk 2]
```diff
@@ -0,0 +1,517 @@
...
+
+.section-img {
+  width: 50%;
+  border-radius: 8px;
+}
+
+.section-content {
+  width: 50%;
+  border-radius: 8px;
+}
```
Comment (Tufahel) [inline assets/style/main.css:209]: - [ ] According to the given template, every `text` item inside the cards should be left aligned. But in your cards, `Canopy` and `button` are not left aligned. Kindly fix this.

image

File: assets/style/main.css [hunk 3]
```diff
@@ -0,0 +1,517 @@
...
+.about-container {
+  display: flex;
+  align-items: flex-start;
+  margin: 30px;
+}
+
+.about-myself {
+  width: 45%;
+  margin: 30px;
+}
```
Comment (Tufahel) [inline assets/style/main.css]: - [ ] According to the given template, every `text` item inside the card should be left aligned. But in your card, `LET'S CONNECT` and `SEE PROJECT` buttons are not left aligned. Kindly fix this.
- [ ] There should not be any space between `My and self`. Kindly fix this issue also.

image

File: assets/style/main.css [hunk 4]
```diff
@@ -0,0 +1,517 @@
...
+  border-radius: 8px;
+}
+
+.grid-item-4 {
+  background-image: url("../images/desktop/contact.png");
+  background-position: 0 0;
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-bottom: 0;
+}
```
Comment (Tufahel) [inline assets/style/main.css]: - [ ] According to the given template, there should be some shapes in the `background image` of the `contact me` section. In your design, there are no shapes. Kindly fix the design.

***Given Design***
image

***Your Design***
image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @osmanwako   :wave: 

Well done. :+1:  
You need to fix some issues before moving to the next project. :running_man: 

## Highlights 🌟

Alt Text

✔️ Linters checks are passing
✔️ Use GitHub Flow
✔️ Meaningful commit messages.


## Required Changes ♻️

_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.
- You can mail me at [Mail](mailto:tufahel96@gmail.com) or ask any question at Slack, `Name: Tufahel`

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 5]
```diff
@@ -60,229 +32,100 @@ After you're finished please remove all the comments and instructions!
...
 
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
-
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (Kamba56) [inline README.md:67]: - [Optional] kindly update the key features section of your readme file. This section should contain the key features of your app.

File: assets/style/main.css [hunk 6]
```diff
@@ -0,0 +1,530 @@
...
+  position: absolute;
+  height: 10px;
+  background-color: #000;
+  width: 25%;
+  border: 5px solid #000;
+  bottom: 10px;
+  left: 40%;
+}
+
+@media (max-width: 768px) {
```
Comment (Kamba56) [inline assets/style/main.css:419]: - [Optional] You are expected to build your portfolio mobile-first, so only desktop styling should be in a media query (@media (min-width: 768px)) and all mobile styling should be your default styling. Consider fixing this, please.

**Review by Kamba56 — APPROVED** [review]
Hi @osmanwako, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏!👏👏👏
## Highlights
- [x] All linters passed!👏👏👏
- [x] Great use of Github workflow!👏👏👏
- [x] Descriptive PR!👏👏👏
- [ ] Great job implementing the desktop version

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**